Util/StructUtils/combineStructFields.m

```diff
@@ -5,20 +5,37 @@
     % the different fieldnames in the argument structures.
     %
     % Inputs:
-
+    %
     % any number of structures
     %
     % Outputs:
     %
     % combined - a structure with all fieldnames of the arguments combined
     %
     % Example:
+    % % basic case
+    % a = struct(); b=a;
     % a.x = 1;
     % b.x = 2;
     % b.y = 3;
     % combined = combineStructFields(a,b);
-    % assert(combined.x == 2);
-    % assert(combined.y == 3);
+    % assert(combined.x==2);
+    % assert(combined.y==3);
+    %
+    % % multi structure case
+    % a = struct(); b=a; c=a;
+    % a.x = 1;
+    % b.x = 2;
+    % b(2).x = 3;
+    % b(3).x = 4;
+    % c.x = 3;
+    % c(2).x = 4;
```

Comment:
So I'm confused about this structure combination. It seems like the order of arguments is important. Why combined(2) == 4 (from the c value) and not  3 (from the b value) ? is it really what you want?

Reply:
Yes, order precedence is exactly what I seek here. This example is here to actually point to this behaviour. The function is "equivalent" of dict.update() in python. It will "append with overwrite" all the arguments fields.

In another words:
```combined(2).x == 4``` because the last structure holding the ```x``` field at position ```(2)``` is the c struct.

Another example:
```
b(4).x = 100;
combined = combineStructfields(c,b,a)
assert(isequal(combined.x,[1,3,4,100]))
```